db/schema.rb

```diff
@@ -300,8 +300,8 @@
     t.string   "guid"
     t.text     "author_signature"
     t.text     "parent_author_signature"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",              null: false
```

Comment:
I suppose `db/schema.rb` has nothing to do with this pull-request, you should remove it ;)